Tests/CommandsTests/TestToolTests.swift

```diff
@@ -30,8 +43,47 @@ final class TestToolTests: XCTestCase {
         XCTAssert(try execute(["--version"]).contains("Swift Package Manager"))
     }
 
+    func testSanitizeNone() throws {
+        fixture(name: "Miscellaneous/SanitizersTest") { path in
+            _ = try execute([], packagePath: path)
+        }
+    }
+
+    func testSanitizeThread() throws {
+        fixture(name: "Miscellaneous/SanitizersTest") { path in
+            let cmdline = ["--sanitize=thread"]
+            XCTAssertThrows(BadCodeException.executionFailure) {
+                try executeTrash(cmdline, packagePath: path)
+            }
+        }
+    }
+
+    // Verifies that giving several options still works.
+    func testSanitizeThreadAddress() throws {
+        fixture(name: "Miscellaneous/SanitizersTest") { path in
+            let cmdline = ["--sanitize=thread", "--sanitize=address"]
+            XCTAssertThrows(BadCodeException.executionFailure) {
+                try executeTrash(cmdline, packagePath: path)
+            }
+        }
+    }
+
+    // Verifies that giving several options still works.
+    func testSanitizeAddressThread() throws {
+        fixture(name: "Miscellaneous/SanitizersTest") { path in
+            let cmdline = ["--sanitize=address", "--sanitize=thread"]
+            XCTAssertThrows(BadCodeException.executionFailure) {
+                try executeTrash(cmdline, packagePath: path)
+            }
+        }
+    }
+
     static var allTests = [
         ("testUsage", testUsage),
         ("testVersion", testVersion),
+        ("testSanitizeNone", testSanitizeNone),
+        ("testSanitizeThread", testSanitizeThread),
+        ("testSanitizeThreadAddress", testSanitizeThreadAddress),
+        ("testSanitizeAddressThread", testSanitizeAddressThread),
```

Comment:
I think these tests will fail to run on the Linux CI :( Maybe we can ask for a toolchain to be installed on the bots.

Reply:
@aciidb0mb3r why would they fail? They don't fail for me if the linux has just swift and clang installed.

Reply:
The linux bot machines do not have swift installed.